app/queries/get_activity_log_events.rb

```diff
@@ -52,6 +54,15 @@ def self.application_choice_audits_filter_sql
     filtered_statuses = ApplicationStateChange::STATES_VISIBLE_TO_PROVIDER - IGNORE_STATUS
     status_transitions_to_include = filtered_statuses.map { |status| "'#{status}'" }.join(', ')
 
-    application_choice_audits_filter + " OR audited_changes::json#>>'{status, 1}' IN (#{status_transitions_to_include})"
+    application_choice_audits_filter += " OR audited_changes::json#>>'{status, 1}' IN (#{status_transitions_to_include})"
+    application_choice_audits_filter + interview_cancelled_application_choice_status_change_sql
+  end
+
+  def self.interview_cancelled_application_choice_status_change_sql
+    jsonb_status_clauses = EXCLUDE_APPLICATION_CHOICE_STATUS_CHANGE.map.with_index do |status, idx|
+      "audited_changes::json#>>'{status,#{idx}}' = '#{status}'"
+    end
+
+    " AND NOT(#{jsonb_status_clauses.join(' AND ')})"
```

Comment:
This method could perhaps be reduced to:

```suggestion
    " AND NOT audited_changes @> '{\"status\": [\"interviewing\", \"awaiting_provider_decision\"]}'"
```

Maybe we could also drop the new constant and the map.

Reply:
Ah yes that's much simpler, I was looking for a way to do an array comparison but hadn't thought of this - thanks :+1: